lib/smileWidget.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
+import 'smileBody.dart';
+import 'smileMouth.dart';
+
+class MainWidget extends StatefulWidget {
```

Comment:
Please make smile face according to design picture.

Reply:
fixed